e2core/auth/access.go

```diff
@@ -72,14 +73,17 @@ func (client *AuthzClient) Authorize(token system.Credential, identifier, namesp
 
 func (client *AuthzClient) loadAuth(token system.Credential, identifier string) func() (*TenantInfo, error) {
 	return func() (*TenantInfo, error) {
-		authzReq, err := http.NewRequest(http.MethodGet, client.location+identifier, nil)
+		ctx, cxl := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cxl()
+
+		authzReq, err := http.NewRequestWithContext(ctx, http.MethodGet, client.location+identifier, nil)
```

Comment:
every http request should have a context attached to it. There is a linter rule (not used in this project yet), but it's generally a good idea to have at least a deadline / timeout context present.

The only difference this change makes is that if se2 (`client.location+identifier` points to se2 in most cases) decides to hold up the response, this request will not hang around more than 10 seconds